### PR TITLE
buildscript for libtrust version v0.0.0-20160708172513-aabc10ec26b7

### PR DESCRIPTION
**Pull request by Sapana-Khemkar**
verified libtrust version v0.0.0-20160708172513-aabc10ec26b7 for ubi8.4

### CODE REVIEW

**Comment by seth-priya** [comment]
@kandarpamalipeddi please review

**Review by kandarpamalipeddi — CHANGES_REQUESTED** [review]
Please add a LICENSE File along with the script.

File: l/libtrust/libtrust_ubi_8.4.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# ----------------------------------------------------------------------------
+#
+# Package        : libtrust
+# Version        : 
+# Source repo    :
```
Comment (kandarpamalipeddi) [inline l/libtrust/libtrust_ubi_8.4.sh:5]: Mention Version as : commit id
Source repo as well.

File: l/libtrust/libtrust_ubi_8.4.sh [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+#!/bin/bash
+
+# set -u
+PACKAGE_NAME=libtrust
+PACKAGE_VERSION="v0.0.0-20160708172513-aabc10ec26b7"
+
+#echo $PACKAGE_VERSION
+IFS='-'
+read -ra PACKAGE_VERSION_SPLIT <<< $PACKAGE_VERSION
+PACKAGE_COMMIT_HASH=${PACKAGE_VERSION_SPLIT[2]}
```
Comment (kandarpamalipeddi) [inline l/libtrust/libtrust_ubi_8.4.sh:27]: Can we replace these lines as:
```
PACKAGE_COMMIT_HASH=`echo $PACKAGE_VERSION | cut -d'-' -f3`
```

File: l/libtrust/libtrust_ubi_8.4.sh (hunk 1 above)
Comment (Sapana-Khemkar) [reply l/libtrust/libtrust_ubi_8.4.sh:5]: done

File: l/libtrust/libtrust_ubi_8.4.sh (hunk 2 above)
Comment (Sapana-Khemkar) [reply l/libtrust/libtrust_ubi_8.4.sh:27]: done

**Review by kandarpamalipeddi — APPROVED** [review]

**Pull request merged**